libs/auth/src/lib/auth.spec.ts

```diff
@@ -0,0 +1,7 @@
+import { auth } from './auth'
+
+describe('auth', () => {
```

Comment:
I'm so confused lol, how is this working? As in like, where are the testing config files coming from? I'm assuming this is Vitest, but like how is this even possible?

Reply:
The configs are coming from `vite.config.ts` file in each libraries, which can later be abstracted into a config library